components/BottomNavigation/src/MDCBottomNavigationBar.h

```diff
@@ -123,6 +123,18 @@ typedef NS_ENUM(NSInteger, MDCBottomNavigationBarAlignment) {
  */
 @property(nullable, nonatomic,copy) UIColor *backgroundColor UI_APPEARANCE_SELECTOR;
 
+/**
+ The top padding of each navigation item when the title is visible. The default value is 8. When the
+ title is not visible there is no padding and the icon is centered vertically.
+ */
+@property(nonatomic, assign) CGFloat itemsTitleVisibleTopPadding;
+
+/**
+ The vertical margin between the icon and label of each navigation item when title is laid out below
+ icon. The default value is 0.
+ */
+@property(nonatomic, assign) CGFloat itemsContentVerticalMargin;
```

Comment:
This appears to be for 2 of the 3 `MDCBottomNavigationBarAlignment` values. Do we want to consider a `itemsContentVerticalMargin:forAlignment:` property instead? I suspect that will be easier to keep straight for ourselves and clients when using these values.

Reply:
Padding is space inside a view, margin is space between two views: https://medium.com/frontendshortcut/margin-vs-padding-c1fc8ea8bfaf

I added just the single value because I wasn't sure if we really wanted to be opening up that horizontal spacing as a param - I default to assuming no unless otherwise requested.

Reply:
I spoke with one of the designers and he thinks it makes sense to allow customization of the horizontal padding between the text/icon for horizontal layouts. It might not be in this PR, but I think we should think about how to more generally support customizing the title/icon layouts.